main.py

```diff
@@ -0,0 +1,74 @@
+from utils.board import Board
+from utils.player import Player
+from utils.movable import Movable
+from create_board import create_board
+
+# Parse the input file
+def parse_input_file():
+    file = open('input.txt', 'r')
+    __input = file.read()
```

Comment:
why are you using __ for all the variables?

Reply:
act as private class variables in python

Reply:
Yup , but these should not be abused. In python using __ is just for the user to understand that it is a private variable/method. So don't use it as it makes the code look ugly and unreadable

Reply:
Ok. I will take care of that from next time.